test/tools/d_do_test.d

```diff
@@ -590,7 +591,10 @@ int tryMain(string[] args)
             default:      envData.ccompiler = "c++"; break;
         }
     }
-    bool msc = envData.ccompiler.toLower.endsWith("cl.exe");
+
+    // can't modify the lines above due to auto-tester's patching
```

Comment:
Can you surround it with comments?
```D
/* CAN'T REMOVE BECAUSE OF AUTOTESTER PATCHING
bool msc = envData.ccompiler.toLower.endsWith("cl.exe");
*/

Reply:
Cool. Removed then :)